utils.js

```diff
@@ -136,3 +142,13 @@ function removeStylesheets(win) {
   }
 }
 exports.removeStylesheets = removeStylesheets;
+
+function installStylesheet(win, uri) {
```

Comment:
To avoid duplication of code, how about we have the `installStylesheets`/`removeStylesheets` code above call these functions.